communications/mongo_models.py

```diff
@@ -0,0 +1,13 @@
+from mongoengine import Document, ListField, StringField, IntField, DateTimeField, ReferenceField
+from datetime import datetime
+
+class Room(Document):
+    participants = ListField(IntField(), required=True)  
+    created_at = DateTimeField(default=datetime.now)
+    updated_at = DateTimeField(default=datetime.now)
+
+class Message(Document):
+    room = ReferenceField(Room, reverse_delete_rule=2) 
```

Comment:
Instead of magic number 2, use the constant `CASCADE`:
```
from mongoengine import CASCADE

room = ReferenceField(Room, reverse_delete_rule=CASCADE)
```